lib/engine/game/g_18_norway/game.rb

```diff
@@ -483,6 +484,17 @@ def check_route_token(route, token)
           raise NoToken, 'Route must contain token' unless token
         end
 
+        def update_cert_limit_to(new_cert_limit)
+          @cert_limit = new_cert_limit
+          @log << "Certificate limit is now #{@cert_limit}"
+        end
+
+        def update_cert_limit
+          nr = @corporations.sum(0) { |corporation| nationalized?(corporation) ? 1 : 0 }
```

Comment:
```suggestion
          nr = @corporations.count(&:nationalized?)
```

Reply:
Whoops, I was too hasty in adding this. I agree that `count` is the right call, but this syntax only works if `nationalized?` is a method of `corporation`.

Fixed in 5f61f40cb864eaefd877492b5838661c8af5a7f6

Reply:
Oops, I must have been half asleep when reviewing this. The version in 5f61f40 looks good to me.